servatrice/src/servatrice.cpp

```diff
@@ -557,10 +585,38 @@ void Servatrice::statusUpdate()
             if(smtpClient->enqueueActivationTokenMail(userName, emailAddress, token))
             {
                 queryDelete->bindValue(":name", userName);
-                servatriceDatabaseInterface->execSqlQuery(queryDelete);
+				queryDelete->bindValue(":email_type", "REG");
+				if (!servatriceDatabaseInterface->execSqlQuery(queryDelete)) {
+					qDebug() << "Failed to remove users that registration emails have been sent to: SQL ERROR";
+				}
             }
         }
 
+		//send forgot password emails
+		query = servatriceDatabaseInterface->prepareQuery("select a.name, b.email, b.token from {prefix}_activation_emails a left join {prefix}_users b on a.name = b.name where type = :email_type");
+		query->bindValue(":email_type", "FORGOTPASS");
+		if (!servatriceDatabaseInterface->execSqlQuery(query)) {
+			qDebug() << "Failed to query users to send forgot password emails for: SQL Error";
+			return;
+		}
+
+		queryDelete = servatriceDatabaseInterface->prepareQuery("delete from {prefix}_activation_emails where name = :name and type = :email_type");
+
+		while (query->next()) {
+			const QString userName = query->value(0).toString();
```

Comment:
Can we refer to the names of the values over an index?
`value("field_name")` instead of `value(0)`?